docs/src/Guides/20 Scales.md

```diff
@@ -203,6 +203,36 @@ bot.grow_scale("Filename_here")
 bot.start("token")
 ```
 
-Finally, for the cool bit of Scales, reloading. Scales allow you to edit your code, and reload it, without restarting the bot.
+Additionally, for the cool bit of Scales, reloading. Scales allow you to edit your code, and reload it, without restarting the bot.
 To do this, simply run `bot.regrow_scale("Filename_here")` and your new code will be used. Bare in mind any tasks your scale
 is doing will be abruptly stopped.
+
+
+Finally, you can also pass any **keyword** arguments into module's `setup` and `teardown` functions, it will also send them into your `Scale.shed` method.
```

Comment:
Am I the only one who got a bit confused by this? I'm *genuinely* asking by the way, it took me a bit to understand but I think that's just me.

Reply:
It's grammatically fragmented. It could be rewritten with clearer wording

Reply:
I understand it, but it's definitely not clean

Reply:
How to make it clearer?

Reply:
https://github.com/Discord-Snake-Pit/Dis-Snek/pull/424#discussion_r845777544